ee/cis/win-10/cis-NON-COMPLETED-policy-queries.yml

```diff
@@ -500,6 +500,110 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Join Microsoft MAPS' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting allows you to join Microsoft Active Protection Service (MAPS), which Microsoft renamed to Windows Defender Antivirus Cloud Protection Service and then Microsoft Defender Antivirus Cloud Protection Service.
+    Microsoft MAPS / Microsoft Defender Antivirus Cloud Protection Service is the online community that helps you choose how to respond to potential threats.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled:
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Microsoft Defender Antivirus\MAPS\Join Microsoft MAPS'
+    Note: This Group Policy path may not exist by default. It is provided by the Group Policy template WindowsDefender.admx/adml that is included with the Microsoft Windows 8.1 & Server 2012 R2 Administrative Templates (or newer).
+  query: |
+    # Recommended registry key does not exist
```

Comment:
This one is stupid. You need to click enabled to click disabled. Then it showed up in the registry...

<img width="920" alt="Screenshot 2023-03-31 at 5 03 38 PM" src="https://user-images.githubusercontent.com/71795832/229229994-9cd074af-55d3-49f6-ac45-ecf1e7657b33.png">
![Uploading Screenshot 2023-03-31 at 5.03.34 PM.png…]()

Reply:
<img width="1159" alt="Screenshot 2023-03-31 at 5 03 34 PM" src="https://user-images.githubusercontent.com/71795832/229230105-7786b968-3bb2-4f7d-b21d-b76dbf2f2377.png">

Reply:
Can you put that in the resolution and QA everything else again before I QA again?